src/main/kotlin/net/pwall/json/schema/parser/JSONReader.kt

```diff
@@ -118,6 +118,11 @@ class JSONReader(val uriResolver: (URI) -> InputStream?) {
         }
     }
 
+    fun readJSON(string: String): JSONValue {
+
```

Comment:
Is this a viewing problem or is there an empty line on line 122?

Reply:
It is the empty line. 

Please feel free to fix it. This is my first time with the contribution.🙏

Reply:
I would but I am not the owner of the repository. We have to wait for pwall567. Great contribution though!